tests/test_device.py

```diff
@@ -69,27 +72,26 @@ def test_send():
 
     TEST_DATA = {'key': 'Hello', 'values': 1234}
 
-    @device_b.on('*/direct-msg')
-    async def callback(event, data):  # pylint: disable=W0612
+    @device_b.on('direct-msg')
+    async def callback(sender, data):  # pylint: disable=W0612
         nonlocal message_received
-        print('device_b got message')
-        assert(event == 'device_a/direct-msg')
-        assert(data == TEST_DATA)
+        assert sender == 'device_a'
+        assert data == TEST_DATA
         message_received = True
 
     @device_a.task
     async def send_msg():  # pylint: disable=W0612
-        print('device_a sending message')
-        await device_a.send('device_b', 'direct-msg', TEST_DATA)
+        with suppress(UnknownPeer):
```

Comment:
What circumstances causes UnkownPeer to be raised? We should probably have a script in `examples` that demos `send` and `request`.

Reply:
It is raised until a peer is available, It's probably feasible to have a timeout.